crmd/membership.c

```diff
@@ -376,13 +378,17 @@ populate_cib_nodes(enum node_update_flags flags, const char *source)
 
         g_hash_table_iter_init(&iter, crm_peer_cache);
         while (g_hash_table_iter_next(&iter, NULL, (gpointer *) &node)) {
-            do_update_node_cib(node, flags, node_list, source);
+            xmlNode *update = NULL;
+            update = do_update_node_cib(node, flags, node_list, source);
```

Comment:
I'm reasonably sure that this part of the patch (and the one below) is wrong.
The result of `do_update_node_cib()` is a reference to an element that was added to `node_list`.
Free'ing it here undoes the effect of calling `do_update_node_cib()`